servo-build-dependencies/ci.sls

```diff
@@ -0,0 +1,14 @@
+{% from 'common/map.jinja' import root %}
+{% from tpldir ~ '/map.jinja' import sccache %}
+
+sccache:
+  file.managed:
+    - name: {{ sccache.destination }}
+    - source: https://s3.amazonaws.com/rust-lang-ci/rust-ci-mirror/{{ sccache.version }}-sccache-{{ sccache.platform }}
```

Comment:
Makes sense - sounds good to me! This is r=me, then. Looks like the travis failure is just atimeout to repo.saltstack.com?